partcad/src/partcad/wrappers/custom_cqgi.py

```
@@ -0,0 +1,619 @@
+
+    If you would prefer to access the underlying model without building it,
+    for example, to inspect its available parameters, construct a CQModel object.
+
+    :param script_source: the script to run. Must be a valid cadquery script
+    :return: a CQModel object that defines the script and allows execution
+    """
+    model = CQModel(script_source)
+    return model
+
+
+class CQModel(object):
```

Ruff (0.8.2)

Class `CQModel` inherits from `object`

Remove `object` inheritance

(UP004)